### ISSUE TITLE
Styles are still loaded even when Jupyterlab's --future-skip-styles-for-disabled flag is used.

### ISSUE THREAD

**Issue by suganya-sk** [jupyter-widgets/ipyleaflet#1103]
TLDR - styles are still loaded even when the lab extension is disabled and Jupyterlab's --future-skip-styles-for-disabled flag is used.

Jupyterlab 3.6.0 recently introduced an option --future-skip-styles-for-disabled to avoid loading styles of disabled extensions. This flag is about to become the default in Jupyterlab 4.0 - see section "Future changes" [here](https://jupyterlab.readthedocs.io/en/stable/extension/extension_migration.html#jupyterlab-3-5-to-3-6). This is a much-anticipated step in addressing the performance issues of Jupyterlab.

However, when I disable jupyter-leaflet and start Jupyterlab 3.6.1 using the flag described above, I can see that the styles of this extension are still loaded onto the page, since the styles are loaded directly here - [link](https://github.com/jupyter-widgets/ipyleaflet/blob/e5f22d124369b6b5acdc4ad5058e9a5bf9355010/js/src/jupyter-leaflet.js#L52-L63) while Jupyterlab's recommendation seems to be to define the location of CSS in package.json - [link](https://jupyterlab.readthedocs.io/en/stable/developer/css.html#file-organization)

Seeing this is incompatible with a Jupyterlab option that is about to become default, can this be fixed please?
